Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-findOne.txt

MongoDB\\Collection::findOne()
==============================

.. meta::

>    :description: Discover how to use `MongoDB\Collection::findOne()` to find a single document in a collection with various query and option parameters.

Reviewer: ```suggestion
   :description: Find a single document in a collection with the MongoDB PHP Library, with various query and option parameters.
```
